src/server/game/Mails/Mail.h

```diff
@@ -211,4 +211,21 @@ struct Mail
     [[nodiscard]] bool IsReturnedMail() const { return checked & MAIL_CHECK_MASK_RETURNED; }
 };
 
+struct ServerMail
+{
+    ServerMail() = default;
+    uint32 id{ 0 };
+    uint8 reqLevel{ 0 };
+    uint32 reqPlayTime{ 0 };
+    uint32 moneyA{ 0 };
+    uint32 moneyH{ 0 };
+    uint32 itemA{ 0 };
+    uint32 itemCountA{ 0 };
+    uint32 itemH{ 0 };
+    uint32 itemCountH{ 0 };
+    std::string subject{ "" };
+    std::string body{ "" };
```

Comment:
this empty default

Reply:
Only need `std::string body{ }` ?

Reply:
```cpp
std::string body
```

Reply:
![image](https://user-images.githubusercontent.com/18329778/163937236-1708ebb5-9986-4e06-99a5-1e66ee339b84.png)